### ISSUE TITLE
Posts not being dispatched to remote inboxes.

### ISSUE THREAD

**Issue by JawadHossain** [CMPUT404F21TEAM/social-distribution#260]
Create a friend with a remote author and try creating a friend post that should be sent to them.
Can use:
https://i-connect.herokuapp.com/inbox
danielcho
123456

or 
https://cmput404-vgt-socialdist.herokuapp.com/
hossain
123456

We are getting 400 Bad request from them.

**Comment by tylerelston** [comment]
I tested cmput404-vgt-socialdist.herokuapp.com and was getting 200 codes after sending a message, although we are getting 404s when accessing /followers/{id} due to them expecting {id} as an actual id and not url. I'll message them about the followers endpoint

**Comment by debecerra** [comment]
Sent post to T20 using our app as well. It was good.

```
2021-12-03T06:37:51.925839+00:00 app[web.1]: [2021-12-02 23:37:51,925] INFO socialDistribution.requests: API POST request to https://cmput404-vgt-socialdist.herokuapp.com/service/author/7db5138f-8027-4de1-8c41-b62ff79887c4/inbox/ and received 201
```

Send a post to T16 with curl because wasn't able to get follows to work with their UI. It was also a 200 code and appears at their GET /inbox route.

Can't find any issues. We can keep this open and keep monitoring as we approach the deadline but seems any problems there may be are on their end.

**Comment by debecerra** [comment]
Found an issue receiving a post. They send this to our inbox which has id of None, which crashes our not none constraint for InboxModels

2021-12-03T06:45:23.004278+00:00 app[web.1]: {'type': 'post', 'title': 'This is a title', 'id': None, 'source': None, 'origin': None, 'description': 'this is a post', 'contentType': 'text/plain', 'content': 'plz send', 'author': {'type': 'author', 'id': 'https://cmput404-vgt-socialdist.herokuapp.com/service/author/7db5138f-8027-4de1-8c41-b62ff79887c4', 'url': 'https://cmput404-vgt-socialdist.herokuapp.com/service/author/7db5138f-8027-4de1-8c41-b62ff79887c4', 'host': 'https://cmput404-vgt-socialdist.herokuapp.com/', 'displayName': 'hossain', 'github': None, 'profileImage': None}, 'categories': ['f'], 'count': 0, 'comments': None, 'published': '2021-12-03T06:45:15.991Z', 'visibility': 'PUBLIC', 'unlisted': True}
